pkg/oauth/providers/github.go

```diff
@@ -44,7 +44,6 @@ func getUserFromToken(c *gin.Context, token *oauth2.Token) (*models.UserCredenti
 	for _, githubUserEmail := range githubUserEmails {
 		if *githubUserEmail.Primary == true {
 			user.Email = githubUserEmail.Email
```

Comment:
Do we need to set UnverifiedEmail = nil or ""?

Reply:
It will automatically be set to nil until we set it othewise